src/Employer/Employer.Web/Filters/LevyDeclarationCheckFilter.cs

```diff
@@ -13,17 +13,18 @@ namespace Esfa.Recruit.Employer.Web.Filters
 {
     public class LevyDeclarationCheckFilter : IAsyncActionFilter, IOrderedFilter
     {
-        private readonly IEmployerVacancyClient _vacancyClient;
         private readonly ILogger<LevyDeclarationCheckFilter> _logger;
         private readonly LevyDeclarationCookieWriter _levyCookieWriter;
+        private readonly IRecruitVacancyClient _recruitVacancyClient;
 
-        public LevyDeclarationCheckFilter(IEmployerVacancyClient vacancyClient,
+        public LevyDeclarationCheckFilter(
             ILogger<LevyDeclarationCheckFilter> logger,
-            LevyDeclarationCookieWriter levyCookieWriter)
+            LevyDeclarationCookieWriter levyCookieWriter,
+            IRecruitVacancyClient recruitVacancyClient)
```

Comment:
The LevyDeclaration check is only applicable on Employer Recruit only shirley? Why is it being put in the Recruit client. Can we get rid of all these client classes (I'm aware I am the **only** one with a problem with them but please please please)

Reply:
This wasn't introduced by me, seems to me like tech debt... but greater tech debt is to get rid of client classes as you mentioned and I share that sentiment.